syntax = "proto3";
enum MessageRetCode {
   MRC_SUCCESSED        = 0;    //成功
   MRC_UNKNOW_ERROR     = 1;    //未知错误
   MRC_DISCONNECT       = 2;    //连接断开
   MRC_ACCOUNT_EXIST    = 3;    //账号己存在
   MRC_NAME_EXIST       = 4;    //角色名己存在
   MRC_ACCOUNT_SEALED   = 5;    //账号被封禁
   MRC_BAD_CLIENT_VER	= 6;    //客户端版本号不正确

   //无效的ID
   MRC_INVALID_ACCNAME   = 100;    //无效的账名名
   MRC_INVALID_ACCID     = 101;    //无效的账号ID
   MRC_INVALID_PASSWORD  = 102;    //无效的密码
   MRC_INVALID_NAME      = 103;    //无效的名字
   MRC_INVALID_ROLEID    = 104;    //无效的角色ID
   MRC_INVALID_ITEMID    = 105;    //无效的物品ID
   MRC_INVALID_COPYID    = 106;    //无效的副本ID
   MRC_INVALID_PETID     = 107;    //无效的宠物ID
   MRC_INVALID_PARTNERID = 108;    //无效的伙伴ID
   MRC_INVALID_EQUIP_ID  = 109;    //无效的装备ID
   MRC_INVALID_CARRERID  = 110;    //无效的职业ID
   MRC_INVALID_SKILL_ID  = 111;    //无效的技能ID
   MRC_INVALID_TARGET_ID = 112;    //无效的目标ID

   //不足的条件
   MRC_NOT_ENOUGH_ACTOIN   = 200;    //行动力不足
   MRC_NOT_ENOUGH_MONEY    = 201;    //货币不足
   MRC_NOT_ENOUGH_ITEM     = 202;    //道具不足
   MRC_NOT_ENOUGH_TIMES    = 203;    //次数不足
   
   //技能
   MRC_SKILL_CD_ERROR      = 300;    //技能未冷却    
   MRC_SKILL_DEAD_OBJ      = 301;    //对象己经死亡
    
}

